packages/view/backend/sirius-components-view-form-edit/src/main/java/org/eclipse/sirius/components/view/form/provider/FormEditPlugin.java

```diff
@@ -90,15 +90,8 @@ protected static String extensionFor(String key) {
         int index = key.lastIndexOf('.');
         if (index != -1) {
             String extension = key.substring(index + 1);
-            if ("png".equalsIgnoreCase(extension) ||
-                "gif".equalsIgnoreCase(extension) ||
-                "bmp".equalsIgnoreCase(extension) ||
-                "ico".equalsIgnoreCase(extension) ||
-                "jpg".equalsIgnoreCase(extension) ||
-                "jpeg".equalsIgnoreCase(extension) ||
-                "tif".equalsIgnoreCase(extension) ||
-                "tiff".equalsIgnoreCase(extension) ||
-                "svg".equalsIgnoreCase(extension)) {
+            if ("png".equalsIgnoreCase(extension) || "gif".equalsIgnoreCase(extension) || "bmp".equalsIgnoreCase(extension) || "ico".equalsIgnoreCase(extension) || "jpg".equalsIgnoreCase(extension)
+                    || "jpeg".equalsIgnoreCase(extension) || "tif".equalsIgnoreCase(extension) || "tiff".equalsIgnoreCase(extension) || "svg".equalsIgnoreCase(extension)) {
```

Comment:
Is this reformatting on purpose?
We could also have something like:
```java
private static final List<String> IMAGE_EXTENSIONS = List.of(...);
...
return IMAGE_EXTENSIONS.stream().anyMatch(supportedExtension -> supportedExtension.equalsIgnoreCase(extension));
```

Reply:
I'll restore the original formatting